PostHog/PostHogStorage.swift

```diff
@@ -38,10 +50,18 @@ func appGroupContainerUrl(config: PostHogConfig) -> URL? {
 
     let bundleIdentifier = getBundleIdentifier()
 
-    let url = FileManager.default
-        .containerURL(forSecurityApplicationGroupIdentifier: appGroupIdentifier)?
-        .appendingPathComponent("Library/Application Support/")
-        .appendingPathComponent(bundleIdentifier)
+    #if os(tvOS)
+        // tvOS: Due to stricter sandbox rules, creating "Application Support" directory is not possible on tvOS
+        let url = FileManager.default
+            .containerURL(forSecurityApplicationGroupIdentifier: appGroupIdentifier)?
+            .appendingPathComponent("Library/Caches/")
+            .appendingPathComponent(bundleIdentifier)
+    #else
+        let url = FileManager.default
+            .containerURL(forSecurityApplicationGroupIdentifier: appGroupIdentifier)?
+            .appendingPathComponent("Library/Application Support/")
+            .appendingPathComponent(bundleIdentifier)
+    #endif
```

Comment:
this can be a method, and we just pass the `appendingPathComponent` as param, so DRY